### PR TITLE
Changed Pillar One story and images

### PR DESCRIPTION
**Pull request by slgill**
Changed the Pillar One story and images.

### CODE REVIEW

File: site/empoweringleaders.html [hunk 1]
```diff
@@ -27,12 +27,25 @@
 
         <p>
           With increased gender training, Peace Corps Volunteers like Alexandra will be able to make an even bigger, more sustainable difference in the lives of young women around the world.
+        </p> -->
```
Comment (cmc333333) [inline site/empoweringleaders.html]: You can go ahead and just delete that content rather than commenting it out.

**Comment by cmc333333** [comment]
Feel free to replace the existing images, too (rather than giving them a suffix). All files are in version control, so you're not going to lose them.

**Comment by seanherron** [comment]
Echoing CM, I would suggest deleting the content rather than commenting it, and not suffixing image files. You should be able to just make those changes and push up another commit to the existing pull request.

**Comment by slgill** [comment]
No problem, the changes have been made. Thanks - Steve

**Comment by cmc333333** [comment]
Very nice. We'll get that to production shortly.

**Comment by seanherron** [comment]
I fixed one issue (https://github.com/18F/letgirlslearn/pull/6) and moved images to the `src` directory rather than the `site` directory so that the build process works correctly, verified in staging, and just uploaded to production. Working to get PC rights to upload to staging and production as well, but it requires AWS credentials.

**Pull request merged**